README.md

```diff
@@ -144,10 +144,11 @@ To build the Open XML SDK
 
 If you want to use a command line approach:
 
-1. Go to the directory that contains the solution.
+1. Go to the directory that contains the project file, i.e. `cd <clone path>\src\DocumentFormat.OpenXml`
```

Comment:
Why are you changing it to the project directory? I always go to the solution directory

Reply:
because I ran 'dotnet build DocumentFormat.OpenXml.csproj' from the solution folder and it complained that the csproj wasn't found. I'll rework and include your fixes below.

Reply:
Oh - no need to include the csproj. Just run `dotnet build`